pkg/kubelet/util/bootime_util_linux_test.go

```diff
@@ -0,0 +1,36 @@
+// +build freebsd linux
+
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package util
+
+import (
+	"testing"
+	"time"
+)
+
+func TestGetBootTime(t *testing.T) {
+	boottime, err := GetBootTime()
+
+	if !boottime.After(time.Time{}) {
+		t.Errorf("Invalid system uptime")
+	}
+
+	if err != nil {
```

Comment:
I think this judgment should be in the first step

Reply:
I think style wise it probably should.

@hongliang5316 Any other ideas on how else you would you improve coverage on `GetBootTime`?

Reply:
Yeah I think style wise it probably should.

@hongliang5316 Any other ideas on how else you would you improve coverage on GetBootTime?

Reply:
+1 I think that moving the nil check first is the most ideomatic.

Reply:
@farah 
I think `GetBootTime` will often return `err == nil`, maybe you can mock `unix.Sysinfo` which inside `GetBootTime`  and make it return `err != nil` to improve coverage